keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -422,20 +431,43 @@ type ClusterNetworkPolicyIngress/EgressRule struct {
 	Name         string
 }
 
-type ClusterNetworkPolicyType string
+// WorkloadSelector selects workloads based on the label Selector.
+// Right now the only selector type supported is PodSelector. More selector types may be
+// added in the future to support other workload selection mechanism, e.g. serviceAccountSelector.
+// Must be accompanied by a NamespaceSelector to specify the scope of the workload selection.
```

Comment:
I think we can remove this statement here.. this is more appropriate when the WorkloadSelector is referenced in the rules